tests/cypress/integration/components/warning-callout.js

```diff
@@ -0,0 +1,28 @@
+describe('warning call out component tests', function () {
+  beforeEach(function () {
+    cy.viewport('macbook-15');
+  });
+
+  it('Navigating to warning callout Component', function () {
+    cy.visit(Cypress.config().baseUrl + '/warning-callout.html')
+  });
+
+  it('Checking for warning callout class', function () {
+    cy.get('.nhsuk-warning-callout')
+  });
+
+  it('Checking for warning callout label class', function () {
+    cy.get('.nhsuk-warning-callout').children().should('have.class', 'nhsuk-warning-callout__label')
```

Comment:
Can this check for content aswell? To make sure that there is content within the label such as 'Important' and not just the class name exists. Because in theory you are just checking the class name exists, not that there is a heading.